.github/workflows/it-transaction.yml

```diff
@@ -42,6 +42,8 @@ on:
       - 'shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/**'
       - 'shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/**'
       - '!*.md'
+  schedule:
+    - cron: '0 21 */1 * *'  # once a day. UTC time
```

Comment:
Suggest to add `if: (github.event_name == 'schedule' && github.repository == 'apache/shardingsphere')` to avoid it's triggered in forked project

Reply:
it's not like the tests in scaling, this should be executed nightly even in forked repository.

Reply:
> Suggest to add `if: (github.event_name == 'schedule' && github.repository == 'apache/shardingsphere')` to avoid it's triggered in forked project

it's not like the tests in scaling, this should be executed nightly even in forked repository.